### ISSUE TITLE
Best practice for including line breaks in long strings?

### ISSUE THREAD

**Issue by JohnMcLear** [wikimedia/jquery.i18n#160]
Just wondering what you guys recommend to do?  I have a lot of TOS / T&C licenses we translate down and I want to keep them as individual strings in my JSON files if poss :)

**Comment by Nikerabbit** [comment]
Splitting per paragraph would be beneficial for translators as well. But you can use \n inside the strings, it just gets very ugly to edit them manually.

**Comment by JohnMcLear** [comment]
``\n`` doesn't work @Nikerabbit - Are you sure ``\n`` is the right approach?  It makes sense but it's weird that it doesn't work!

**Comment by Nikerabbit** [comment]
Are you expecting the newlines to appear in rendered HTML? That won't work. You need to add `<br>`s or `<p>` wrappings to the string or do something like `$dom.append( htmlescape( $.i18n( key ) ).replace( / /\n/, '<br>' ) )`.

Or just split the paragraphs in the file :)

**Comment by JohnMcLear** [comment]
Yes I was hoping the newlines to appear in rendered HTML..  I'm being totally dumb here..  Can you provide a HTML / JSON / JS file showing example?  I tried adding <br> to the string and no dice with that..  I can't imagine I have to do a replace on language change, that seems far too clunky just to add line breaks!

**Comment by Nikerabbit** [comment]
It is very clunky indeed, hence my persistent recommendation to use separate messages per paragraph :)

**Comment by Mohd-Avesh** [comment]
By using Sequence characters we can add line breaks (e.g. \n,,,)
